### PR TITLE
Add missing code examples on slice iter types

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Dylan-DPC

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Fixed!

File: library/core/src/slice/iter.rs [hunk 1]
```diff
@@ -270,6 +270,13 @@ pub(super) trait SplitIter: DoubleEndedIterator {
 ///
 /// This struct is created by the [`split`] method on [slices].
 ///
+/// # Example
```
Comment (pickfire) [inline library/core/src/slice/iter.rs:273]: This should be `Examples` for consistency, not `Example`.

Comment (GuillaumeGomez) [reply library/core/src/slice/iter.rs:273]: Ah true. I might have make the same mistake in some other places...

**Pull request merged**